### ISSUE TITLE
Blank screen is displayed after creating a new @Expensifail.com domain account,

### ISSUE THREAD

**Issue by isagoico** [Expensify/App#4231]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Create a new @Expensifail.com domain account. 
2. Go through the account creation flow 

## Expected Result:
After setting a password, the user should be redirected to the Concierge chat and NewDot main page. 

## Actual Result:
User is redirected to a blank screen and there's a console error. 

## Workaround:
Refreshing solves the issue. 

## Platform:
Where is this issue occurring?

- [x] Web
- [ ] iOS
- [ ] Android
- [ ] Desktop App
- [ ] Mobile Web

**Version Number:** 1.0.80-0

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** 

![image](https://user-images.githubusercontent.com/44479856/127044621-f3a3bff3-4659-42c3-b174-36f109a6eee6.png)

![image](https://user-images.githubusercontent.com/44479856/127044650-6876401c-1f5f-4853-be52-db4b002ff995.png)

**Expensify/Expensify Issue URL:**

[View all open jobs on Upwork](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2)

___ 
From @vitHoracek https://expensify.slack.com/archives/C01GTK53T8Q/p1627293275255500

> After creating a new @expensifail.com account on STAGING, verifying and then logging in with newly setup password, I got blank screen with this error. Refresh has fixed it. Seems like some data might not be fetched on first try correctly.

**Comment by MelvinBot** [comment]
Triggered auto assignment to @MonilBhavsar (`Engineering`), see https://stackoverflow.com/c/expensify/questions/4319 for more details.

**Comment by MonilBhavsar** [comment]
@vitHoracek, I am not able to reproduce this, Can you please check again if it wasn't a temporary error

**Comment by mountiny** [comment]
@MonilBhavsar I will try again, although it happened to me twice, whenever there was nothing stored in localStorage, maybe try to do that first.

**Comment by mountiny** [comment]
I have tried it again and could not replicate, however this time, I have used `vit+2@expensifail.com` email and after signin in for a first time, I saw a chat which account `vit+1@expensifail.com` had with different account, but with Auth Get errors for the request. I have cleared all app memory before so it is odd.

But I could not replicate this original problem again, so it seems that both are just very odd issues and we can probably close this and see if it happens again.
